crate/pages/docs/faq.html

```diff
@@ -1,71 +1,72 @@
-title: Frequently Asked Questions
-description: Frequently Asked Questions about Crate
+title: Frequently Asked Questions description: Frequently Asked Questions about Crate
```

Comment:
Ah yes, it's because these files aren't 'real' markdown, but my editor thinks they are and so 'corrects' them. Hang on, will fix up now.